WordPress/gutenberg
# Framework: Remove packages build from build, dev scripts

Previously: #15219
Related: #14560

This pull request seeks to update the Gutenberg-specific Webpack configuration to avoid running Babel as part of the Webpack build. As described at https://github.com/WordPress/gutenberg/pull/15219#issuecomment-487133283, the files processed by Webpack are _already transpiled_ by Babel as a step immediately prior. Including `babel-loader` is therefore redundant, wasted effort.

With this change, I'd measured approximately a **57% decrease** in the time it takes to run `npx wp-scripts build` (~16.7s to 7.1s, measured using [`time`](https://linux.die.net/man/1/time)).

**Testing Instructions:**

1. Reset your repository using `git clean -fdx` (**Warning:** This will forcefully remove any uncommitted files)
2. Run `npm run build`
3. Verify that JavaScript outputs both in `build/*/*.js` and in `packages/*/build/*.js` are transpiled as expected by Babel (e.g. no ES2015 or JSX)
